framework/src/st_things/st_things.c

```diff
@@ -558,6 +634,10 @@ st_things_representation_s *st_things_create_representation_inst()
 	return rep;
 }
 
+//--------------------------------------------------------------------------------
+/**
+ * Function definition for deleting an instance of representation.
```

Comment:
Will you use `.` or not at end of line? Some line does not have, some line has.

Reply:
done.